src/StatusBar.ts

```diff
@@ -127,6 +127,9 @@ export class StatusBar {
   }
 
   private get activeFolder() {
+    if (!this._activeFolder && vscode.workspace.workspaceFolders.length === 1) {
+      this._activeFolder = vscode.workspace.workspaceFolders[0].name
+    }
     if (!this._activeFolder && vscode.window.activeTextEditor) {
       const folder = vscode.workspace.getWorkspaceFolder(vscode.window.activeTextEditor.document.uri)
       if (folder) {

```

Comment:
maybe combine these 2 conditions under `if (!this._activeFolder) {...}` to reduce a bit mental overhead? Also, did you mean to have an `else` after the first condition?

Reply:
Either there have to be two `if(!this._activeFolder)` or an `else`. I'll change it.